torch/library.py

```diff
@@ -452,7 +452,7 @@ def _del_library(
     op_defs,
     registration_handles,
 ):
-    import torch.fx
+    from torch.fx.operator_schemas import _SCHEMA_TO_SIGNATURE_CACHE
```

Comment:
Ugh i think the problem is that the finalizer isn't allowed to import anything, because it might be run when python is exiting.

If that's the case _del_library should take _SCHEMA_TO_SIGNATURE_CACHE as an input and then mutate it.

Reply:
Another reason why it should take that variable is that it might be gone by the time the finalizer runs too

Reply:
this is a good catch. updating